src/test/test-mount-util.c

```diff
@@ -538,9 +538,44 @@ TEST(bind_mount_submounts) {
 }
 
 TEST(path_is_network_fs_harder) {
-        ASSERT_OK_ZERO(path_is_network_fs_harder("/dev"));
-        ASSERT_OK_ZERO(path_is_network_fs_harder("/sys"));
-        ASSERT_OK_ZERO(path_is_network_fs_harder("/run"));
+        int r;
+
+        ASSERT_OK(path_is_network_fs_harder("/"));
+        ASSERT_OK(path_is_network_fs_harder("/dev"));
+        ASSERT_OK(path_is_network_fs_harder("/sys"));
+        ASSERT_OK(path_is_network_fs_harder("/run"));
```

Comment:
Please remove these assertions, access might be blocked in a myriad of ways or they might be something completely different and unexpected